zhaquirks/sonoff/zbminir2.py

```diff
@@ -0,0 +1,88 @@
+"""Sonoff ZBMINIR2 - Zigbee Switch."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class SonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        external_trigger_mode = ZCLAttributeDef(
+            id=0x0016,
+            type=t.uint8_t,
+        )
+        detach_relay = ZCLAttributeDef(
+            id=0x0017,
+            type=t.Bool,
+        )
+        turbo_mode = ZCLAttributeDef(
+            id=0x0012,
+            type=t.int16s,
+        )
+
+    async def _read_attributes(
+        self,
+        attribute_ids: list[t.uint16_t],
+        *args,
+        manufacturer: int | t.uint16_t | None = None,
+        **kwargs,
+    ):
+        """Read attributes ZCL foundation command."""
+        return await super()._read_attributes(
+            attribute_ids,
+            *args,
+            manufacturer=foundation.ZCLHeader.NO_MANUFACTURER_ID,
+            **kwargs,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
+
+
+class SonoffExternalSwitchTriggerType(types.enum8):
+    """extern switch trigger type."""
+
+    edge_trigger = 0x00
+    pulse_trigger = 0x01
+    normally_off_follow_trigger = 0x02
+    normally_on_follow_trigger = 0x82
```

Comment:
Renaming these to have the first letter upper-case for now. There's no real "standard" for this yet.
```suggestion
    Edge_trigger = 0x00
    Pulse_trigger = 0x01
    Normally_off_follow_trigger = 0x02
    Normally_on_follow_trigger = 0x82
```